index.html

```diff
@@ -0,0 +1,148 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://use.typekit.net/mxx0rob.css">
+  <link rel="stylesheet" href="style.css">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous">
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-u1OknCvxWvY5kfmNBILK2hRnQC3Pr17a+RTT6rIHI7NnikvbZlHgTPOOmMi466C8" crossorigin="anonymous"></script>
+  <script src="https://kit.fontawesome.com/bf66e12569.js" crossorigin="anonymous"></script>
+  <title>Program</title>
+</head>
+
+<body class="body">
+  <header class="header">
+    <div id="hide-div"><i class="fa-solid fa-bars menu-bars"></i></div>
+    <div class="mobile-menu">
+      <i class="fa-solid fa-x menu-x"></i>
+      <ul class="menu-mobile-list unic-font">
+        <li class="menu-mobile-list-itens"><a href="index.html" class="no-underscore color-three">Home</a></li>
+        <hr class="menu-mobile-line">
+        <li class="menu-mobile-list-itens"><a href="about.html" class="no-underscore color-three">About</a></li>
+        <hr class="menu-mobile-line">
+        <li class="menu-mobile-list-itens color-three"><a href="#" class="no-underscore color-three">Program</a></li>
+        <hr class="menu-mobile-line">
+        <li class="menu-mobile-list-itens color-three"><a href="#" class="no-underscore color-three">Join</a></li>
+        <hr class="menu-mobile-line">
+        <li class="menu-mobile-list-itens color-three"><a href="#" class="no-underscore color-three">Sponsor</a></li>
+        <hr class="menu-mobile-line">
+        <li class="menu-mobile-list-itens color-three"><a href="#" class="no-underscore color-three">News</a></li>
+      </ul>
+    </div>
+    <div class="desktop-menu">
+      <ul class="desktop-list color-three">
+        <i class="fa-brands fa-facebook-f"></i>
+        <i class="fa-brands fa-twitter"></i>
+        <li class="desktop-list-itens">English</li>
+        <li class="desktop-list-itens">My Page</li>
+        <li class="desktop-list-itens">Logout</li>
+      </ul>
+      <ul class="menu-desktop-list">
+        <a href="index.html"class=""><img src="./assets/logo/image.jpg" alt="bgs logo" class="bgs-logo"></a>
+        <li class="menu-desktop-list-itens"><a href="about.html" class="no-underscore color-one">About</a></li>
+        <li class="menu-desktop-list-itens color-one"><a href="#" class="no-underscore color-one">Program</a></li>
+        <li class="menu-desktop-list-itens color-one"><a href="#" class="no-underscore color-one">Join</a></li>
+        <li class="menu-desktop-list-itens color-one"><a href="#" class="no-underscore color-one">Sponsor</a></li>
+        <li class="menu-desktop-list-itens color-one"><a href="#" class="no-underscore color-one">News</a></li>
+      </ul>
```

Comment:
### You did a great job, your home page looks like the design provide but in the desktop version there is no way to come back into your home page please add the link to your home page on your about page 
below I provide clear information please make sure you implement it as it required [ here ](https://github.com/PedroH-Neves/Capstone-project-1/pull/1#discussion_r978232877) by the previous reviewer:grinning:  :grinning:  :grinning: 
- [ ] Kindly make sure your project contains a `Home` page and `About` page.
- [ ] It's possible to navigate between the pages, there's a link from the `Home` page to the `About` page and the opposite too. it means on your home page you will have the link to your about page and on the about page, you will have the link to your home page 
- [ ] Both pages contain the same navbar and footer.

Check the  [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) to see the style and the container of your about page :+1: :grinning: :grinning: :grinning: